src/Xamarin.Android.Build.Tasks/Utilities/MarshalMethodsAssemblyRewriter.cs

```diff
@@ -95,10 +123,161 @@ void MoveFile (string source, string target)
 				Files.CopyIfChanged (source, target);
 				try {
 					File.Delete (source);
-				} catch (Exception ex) {
+				} catch (Exception) {
 					log.LogWarning ($"Unable to delete source file '{source}' when moving it to '{target}'");
 				}
 			}
+
+			string ToStringOrNull (object? o)
```

Comment:
I totally expected this method to "normalize" the empty string to `null`, not what this method *actually* does.

Also, given that this appears to be for "debug" output, why is it `'null'` (with single-quotes)?  Wouldn't outuput of:

```
method.CallbackField == null
```

make more sense than the current:

```
method.CallbackField == 'null'
```

?

Reply:
This method will go away in the next PR in the series.